lib/beaker/options/presets.rb

```diff
@@ -110,7 +110,7 @@ def env_vars
       def presets
         h = Beaker::Options::OptionsHash.new
         h.merge({
-          :project                => 'Beaker',
+          :project                => ENV['JOB_NAME'] || 'Beaker',
```

Comment:
Pulling environment variables in here may be somewhat confusing for people reading the code since there is another place where we pull in environment variables:
https://github.com/sschneid/beaker/blob/bkr-161_tagging/lib/beaker/options/presets.rb#L16

If you pull in JOB_NAME there you can be more explicit about which variables take precedence. If I understand correctly, setting it here will definitely allow it to be overridden by BEAKER_project or BEAKER_PROJECT.

However, I am not generally familiar with recent Beaker development goals and I see that other variables are pulled in within this code block so *shrug*.